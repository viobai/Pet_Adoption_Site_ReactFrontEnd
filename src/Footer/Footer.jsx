import React from 'react';
import style from "./Footer.module.css";

const Footer = () => {
    return (
        <footer className={style.footerContainer}>
            <div>
                <h3>Contact Us</h3>
                <p>Phone: (XXX) XXX-XXXX</p>
                <p>Email: <a className={style.email} href={`mailto:rescuePrjHumane@placeholder.com`}>rescuePrjHumane@placeholder.com</a></p>
                <p>#XXX XX-th Ave NE, Seattle, WA 98105</p>
            </div>
            <div>
                <h3>Adoption Hours</h3>
                <p>Mon - Thu: 12 a.m. - 6 p.m.</p>
                <p>Fri - Sun: 10 a.m. - 8 p.m.</p>
            </div>
        </footer>
    )
}

export default Footer
