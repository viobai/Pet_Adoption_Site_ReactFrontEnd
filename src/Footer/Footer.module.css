:root {
    --watermelonRed:  rgb(197, 47, 27);
}

.footerContainer {
    display: flex;
    padding: 1rem 0 2.5rem 2rem;
    background-color: #f0eae76c;
}

.footerContainer > * {
    margin-right: 2rem;   
}

.footerContainer > div > * {
    margin: 0.5rem;
    padding: 0rem;
    color: var(--watermelonRed);
}

.footerContainer > div > p {
    margin: 0.5rem;
    padding: 0rem;
    color: var(--watermelonRed)
}

.email {
    color: var(--watermelonRed);
}

.email:hover {
    color: rgba(31, 31, 247, 0.836);
}
