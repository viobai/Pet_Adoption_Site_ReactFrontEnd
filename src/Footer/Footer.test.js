 
import React from "react";
import Footer from "./Footer";

describe("Footer tests", () => {
    it("renders correctly", () => {  
        const { container } = render(<Footer/>);
        expect(container).toMatchSnapshot();
    });
});