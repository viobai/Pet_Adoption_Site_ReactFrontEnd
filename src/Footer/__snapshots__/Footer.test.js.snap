// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests renders correctly 1`] = `
<div>
  <footer
    class="footerContainer"
  >
    <div>
      <h3>
        Contact Us
      </h3>
      <p>
        Phone: (XXX) XXX-XXXX
      </p>
      <p>
        Email: 
        <a
          class="email"
          href="mailto:rescuePrjHumane@placeholder.com"
        >
          rescuePrjHumane@placeholder.com
        </a>
      </p>
      <p>
        #XXX XX-th Ave NE, Seattle, WA 98105
      </p>
    </div>
    <div>
      <h3>
        Adoption Hours
      </h3>
      <p>
        Mon - Thu: 12 a.m. - 6 p.m.
      </p>
      <p>
        Fri - Sun: 10 a.m. - 8 p.m.
      </p>
    </div>
  </footer>
</div>
`;
